www/tolo00/cv02/view/BusinessCardBack.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+class BusinessCardBack
+{
+
+    public static function render(Person $person)
+    {
+        echo '
+            <div class="page back-page">

```

Comment:
takhle se HTML v PHP nerenderuje, staticky HTML text musi jit mimo prikaz `echo`

Reply:
@nvbach91 problém jsem opravil, je to takto nyní OK?